### DISCUSSION TITLE
Economy Health Management - AHBot?

### DISCUSSION THREAD

**Discussion by Vyperhand** [chromiecraft/chromiecraft#5451] [Suggestions]
New player here.  Looking at the AH, there is a dearth of low-level usable and crafting items, and the rare things that are posted are definitely posted at prices only an alt could pay.

As a method of improving the health of the economy for new players and to thereby help grow the server and retain said new players, perhaps considering some implementation of AHBot might help?  Even if it's tweaked to only work up to a certain level, if the endgame economy is more healthy/stable than the introductory levels.

A sample module for Azerothcore can be found here:
https://github.com/azerothcore/mod-ah-bot

**Comment by Ildourol** [comment]
Trying to buy something as a low level character is a nightmare. Also, what's the stage in the after stage 70s? Do we have to give a kidney in order to buy enchants and gems?

**Comment by Vyperhand** [comment]
I think partially caused by the influx of challenge characters unable to use the AH, the issue with the economy while leveling has gotten dramatically worse.   Gear and crafting materials for classic/"old world" content are frequently totally unavailable.  There's a demand market in place as I am able to sell world drops reasonably well, but there are too few sellers to keep the market healthy.  Supply needs to be increased somehow until we recover a larger leveling population of non-challenge characters.    There are entire level brackets without gear, the same 2-3 sellers are the only one posting bags, etc.  Even glyphs which are the best-stocked category are posted by very few different players.

No hate, just love and want to see the server have the tools in place to support growth with the hope that heroics and 2x quests get us an influx of even newer blood than me.  🙂

**Reply by Aethereus-ManaOnion** [reply]
If there is ample demand and short supply then you can make tons of bank providing resources to fill the demand

**Reply by Vyperhand** [reply]
Appreciate the vote of confidence, but while I may be good, I'm not stabilize an entire server economy alone good.  ;) In the meantime, makin' profits and doing what can be done.  I can sell all day, but buying remains challenging to impossible just because stuff's not there to buy.

**Comment by Vyperhand** [comment]
To expand on this a little - during a discussion of the AH economy in world, the sample key market of L40 plate was discussed.  At the time, there were a total of 13 L40 plate items on the AH.  The market is just too constricted and needs a gentle supply-side infusion.  Gold will sort itself out for players, and worst case the bot's buying can also be tuned slightly to keep the market moving and prices less volatile.

The same items are showing fluctuation of over 100% weekdays vs. weekends even right now - things like the ubiquitous Traveler's Backpack range from 8g to 17g and back depending on the day.

White world drop items for JC, Tailoring, Alchemy etc.  are minimal or non-existent.  On the rare cases they are posted, the cost tends to run wildly outside a "normal" price range.  Some are consistently posted at or near vendor, while others are blasted to heights no one on-level could reasonably pay even with the gold CC provides for leveling.

For example, Troll Mojo, an item JC need 60 of to grind through a rough stretch of limited orange recipes - when there are any posted, (which is not often), they are priced from 2-4g/each.

This is being exacerbated by a chunk of the leveling community being on challenges that prohibit AH interaction, disrupting both sales and purchases.

I know there's not much enthusiasm for this, but if we don't do something we're going to keep losing new players who don't have the patience to farm everything they need/want because buying it isn't a viable option.

**Reply by Aethereus-ManaOnion** [reply]
Grind Troll Mojo and make billions

**Comment by siorse** [comment]
I had a bank full of JC mats from levelling. I've been trying to sell some on the AH, but a lot don't. Some I have ended up vendoring after trying the AH so many times, simply as I need the bank space now. I've advertised in world chat a few times about having low level JC mats, but no interest. I've had some of these things in my bank for so long.

Maybe something like a bulletin board on the chromie website might help? If someone has a lot of low level mats left over they can mention it on there, if there's interest then they can sell? The discord server trade channel seems to only have level 70 stuff.

As for items - I often try to give blues away in world free for levellers, but I only come across 60+ BoE blues now

On needing mojo - I ended up using other recipes, there was 1 on the AH when I was at that stage and it was 40g. I can't remember which items I crafted in the end but I think I got a good deal on some gems instead. I found on Chromie it is best to try and do it your own way rather than just following the online guides, the economy is different and the people are different. World chat has a lot of generous people too so whilst a recipe may look too expensive to make for levels, or the guides say it will be, you may be surprised on chromie.

**Reply by Aethereus-ManaOnion** [reply]
very good take
